{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -26,6 +29,18 @@ docs: prepare_docs_folder requirements
 docs_check_external_links: prepare_docs_folder requirements
 	sphinx-build -b linkcheck ./docs ./docs/_build
 
+## Run code coverage
+coverage: requirements
+	coverage run -m pytest
+
+## Run code coverage, and produce a HTML output
+coverage_html: coverage
+	coverage html
+
+## Run code coverage, and produce an XML output
```

Comment:
Why XML (over JSON, for instance)?

Reply:
I think I added this in case users wanted to use [Codecov](https://about.codecov.io/) - unless you install the additional package `pytest-cov`, I think you need an XML file to upload coverage reports - see [here](https://github.com/codecov/example-python#faq), although I do note that Codecov should [support JSON](https://docs.codecov.io/docs/supported-report-formats) (as well as other formats).